src/server/acl/user.h

```diff
@@ -66,6 +78,9 @@ class User final {
   // its respective ID within the commands vector.
   static size_t Selector(std::string_view);
 
+  AclKeys Keys() const;
+  const AclKeys& KeysRef() const;
```

Comment:
Why not just have `Keys()` return a reference and whoever needs to copy it will just do so? 🙂 I don't know any framework or library that uses such accessors

Reply:
const ref.